src/ls.js

```diff
@@ -128,3 +149,16 @@ function _ls(options, paths) {
   return list;
 }
 module.exports = _ls;
+
+
+function ls_stat(path) {
+  var stats = fs.statSync(path);
+  // Note: this object will contain more information than .toString() returns
+  var long_stats = Object.create(stats);
+  long_stats.name = path;
```

Comment:
Ah, I see. In that case `path` or `relativePath` would be more semantic than `name`, but only a by a little bit, so close enough! :wink: